src/parachain/system.ts

```diff
@@ -52,4 +60,9 @@ export class DefaultSystemAPI implements SystemAPI {
         const head = await this.api.rpc.chain.getFinalizedHead();
         return await this.api.query.security.parachainStatus.at(head);
     }
+
+    async setCode(code: string): Promise<void> {
```

Comment:
Maybe add a comment here that this delegates through `sudoUncheckedWeight`, otherwise it is misleading.